api/app/config_temp.py

```diff
@@ -1 +1,24 @@
-DATABASE_URL = 'postgres+psycopg2://<USERNAME>:<PASSWORD>@<IP_ADDRESS>:<PORT>/<DATABASE_NAME>'
\ No newline at end of file
+DATABASE_URL = 'postgres+psycopg2://<USERNAME>:<PASSWORD>@<IP_ADDRESS>:<PORT>/<DATABASE_NAME>'
+
+SEED_DATA = True    # Whether to clear the current database tables and add users
+TEST_SERVER = True  # If database is being seeded, decides whether to create 
+                    # example elections and questions
+
+MEMBERSHIP_API = '' # Host Port for the membership portal 
+
+headers = {
+  'Content-Type': 'application/json'
+}
+payload = {
+    'email':'', # Membership Portal account email
+    'password': '' # Membership Portal account password
+}
+
+# Membership Portal Login
+token_response = requests.post(MEMBERSHIP_API + 'api/v1/auth/login', data = json.dumps(payload), headers=headers)
+token_json =  token_response.json()
+
```

Comment:
Configuration files should _generally_ not have any functions running in them, since they're supposed to be immutable. It's best we remove the call to the API login and simply just add the JWT to the configuration file manually after querying the portal ourselves.
That being said, the TOKEN environment variable itself is better off being a simple string, rather than a dict for authentication. This is because it's easier to reuse the variable later on, should we need it in other contexts rather than this one.
To know what value should be in a configuration file, think of it this way: "Can I put this as an environment variable?" (number, string, etc.) If yes, you're golden.